src/flamegpu/io/JSONLogger.cu

```diff
@@ -277,12 +277,12 @@ void JSONLogger::logCommon(const RunLog &log, const RunPlan *plan, bool doLogCon
     rapidjson::StringBuffer s;
     if (prettyPrint) {
         // rapidjson::Writer doesn't have virtual methods, so can't pass rapidjson::PrettyWriter around as ptr to rapidjson::writer
-        rapidjson::PrettyWriter<rapidjson::StringBuffer>* writer = new rapidjson::PrettyWriter<rapidjson::StringBuffer>(s);
+        auto writer = new rapidjson::PrettyWriter<rapidjson::StringBuffer, rapidjson::UTF8<>, rapidjson::UTF8<>, rapidjson::CrtAllocator, rapidjson::kWriteNanAndInfFlag>(s);
```

Comment:
Ah so this is the secret template args to enable NAN, I couldn't even figure out a combination that built the one or two times I looked into it.

Reply:
Yeah the docs are not particularly clear on this.